### PR TITLE
Fixed the scroll up button layout issue

### PR DESCRIPTION
**Pull request by linkfang**
This is just for testing and preview, the actual fix/PR is here:
https://github.com/EclipseFdn/solstice-assets/pull/275

### CODE REVIEW

**Comment by autumnfound** [comment]
One thing I'm not 100% sure of, on netlify there is no chatbot active so the pointer is oddly placed. Is there ever a case on production where that would not be present?

**Review by chrisguindon — CHANGES_REQUESTED** [review]
As discussed, we should only move the .scrollup if the chatbox is present on the page: https://legacydocs.hubspot.com/docs/methods/conversations_api/hubspot-conversations-javascript-api

**Comment by chrisguindon** [comment]
I will ping @shanda-eclipse to see if we can figure a way to include the chatbox on staging/netlify.

File: src/main/www/src/less/App.less [hunk 1]
```diff
@@ -289,6 +289,8 @@ html {
 
 .scrollup {
   cursor: pointer;
+  // bottom: 100px;
```
Comment (autumnfound) [inline src/main/www/src/less/App.less:292]: we should remove these lines if we're on the final version ;)

Comment (linkfang) [reply src/main/www/src/less/App.less:292]: Ah, no, this PR is just for testing and preview, the actual changes are in solstice-assets repo (https://github.com/EclipseFdn/solstice-assets/pull/275)

**Comment by linkfang** [comment]
Closing this PR as the actual patch was merged.

**Pull request closed without merging**